.github/workflows/Build-Rock.yaml

```diff
@@ -1,206 +1,227 @@
-name: Build rock
+name: Build Rock
```

Comment:
Are you sure you want to change it? Usually acronyms are spelled out in all uppercase (AFAIK for example), but since we are just using it as a name, title case should be fine.

Reply:
I don't mind either way, but changing it here will mean you need to change it everywhere across the repo and the documentation, to keep the consistency. You better consult with Cris about that because he is the one who changed "ROCK" to "rock" a couple of months ago.